freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -677,8 +694,8 @@ def _create_original_to_rebuilt_nvrs_map(self):
                 # all of them, that have overlapping package names
                 for block_build in blocking_advisories_builds:
                     block_build_nvr = parse_nvr(block_build)
-                    if block_build_nvr['name'] == parsed_build_nvr['name'] and \
-                            block_build_nvr['version'] == parsed_build_nvr['version']:
+                    if block_build_nvr['name'] == parsed_build_nvr['name']\
+                            and block_build_nvr['version'] == parsed_build_nvr['version']:
```

Comment:
Per PEP8, wrapping in parentheses should be used in preference to use a backslash for line continuation.

```suggestion
                    if (
                        block_build_nvr["name"] == parsed_build_nvr["name"]
                        and block_build_nvr["version"] == parsed_build_nvr["version"]
                    ):
```